### PR TITLE
Fixes credentials dialog isn't modal

### LINKED ISSUE #2871: Credentials dialog isn't modal
When it appears, user can switch back to VS without closing the dialog and interact with UI

### PR DESCRIPTION
**Pull request by karthiknadig**
Fixes #2871

### CODE REVIEW

File: src/Package/Impl/Shell/VsAppShell.cs [hunk 1]
```diff
@@ -88,6 +86,7 @@ public sealed class VsAppShell : IApplicationShell, IMainThread, IIdleTimeServic
         }
 
         private void OnApplicationStarted(object sender, EventArgs e) {
+            _appConstants.Initialize();
```
Comment (MikhailArkhipov) [inline src/Package/Impl/Shell/VsAppShell.cs]: This only comes when VS was still starting. You need to add this to

```
           if (value is bool) {
                if ((bool)value) {
                    Started?.Invoke(this, EventArgs.Empty);
```

Comment (karthiknadig) [inline src/Package/Impl/Shell/VsAppShell.cs]: I did not see `CheckVsStarted` till i rebased. will update.

**Comment by karthiknadig** [comment]
@MikhailArkhipov does this look good. I will merge if there are no more comments.

**Comment by MikhailArkhipov** [comment]
👍

**Pull request merged**